VIDEO_6_SCRIPT.md

```diff
@@ -0,0 +1,54 @@
+
+## Examples of tools that promote transparency in project work
+
+As mentioned above, the project is managed by all the people who actively collaborate in its community. AsyncAPI has several tools that ensure transparency in project management. Let's talk about them:
+
+- **Open governance**: To ensure that the project is managed by its community and that it’s the one pulling the strings, last year, it was hosted by an open government foundation, Linux Foundation. Hosting the project on a neutral foundation ensures true democratization for the project and its community. At the same time, it provides the long-term sustainability and success of the initiative. 
+
+- **Economy**: all the financial management of the project is managed through Open Collective. It’s an online crowdfunding platform for communities that provides the necessary tools to raise and share the finances raised with complete transparency. You can find out where the money comes from in this space and its use.
+
+- **Transparent communication and work**: To ensure that the principle of transparency governs communication, everything is said and done in public. The community uses Slack as the first communication platform, and secondly, GitHub is used as a means of communication and work. Therefore, all information about any project management is public, no matter how insignificant or essential it may be.
+
+- **Meetings**: There are different meeting points where the community can meet, propose or discuss any topic related to AsyncAPI. Any interested person can join these meetings and raise any thoughts. 
+
+  Currently, there is a bi-weekly general meeting, SIG Meeting, where the most significant changes of the project so far are presented and updated. Then there are three types of more specific arrangements: Contributor First, aimed at informing people who want to contribute, Spec 3.0 Meeting, where the progress and work of the version of the project are discussed. Spec 3.0 Meeting, where the project is concerned, and Spec 3.0 Meeting, where the project is discussed.
+
+- **Decision-making**: Small decisions are usually discussed on GitHub, Slack, or in meetings. For higher level, decision making or when the maintainers cannot find consensus, the Technical Steering Committee (TSC) is responsible for the oversight of the AsyncAPI Initiative. 
+  The Technical Steering Committee (TSC) provides oversight of the AsyncAPI Initiative. 
+
+  Anyone with interest can become a member of the TSC. All you have to do is become a maintainer of one of the AsyncAPI projects. To become a maintainer, regularly contribute to one of the projects, and then other maintainers will invite you to join. 
```

Comment:
this is an explanation that should imho go to Open Governance video, this sentence and the following ones as it explains how our open governance works at asyncapi